extension/webpack.extension.js

```diff
@@ -18,6 +21,22 @@ const prodConfig = (env = { PRODUCTION: false }) => ({
     new webpack.DefinePlugin({
       DEV_SERVER: false,
     }),
+    ...(env.TRANSLATIONS
+      ? [
+          new I18nextWebpackPlugin({
+            async: true,
+            dest: "./",
+            extensions: [".ts", ".tsx"],
+            options: {
+              createOldCatalogs: false,
+              locales: LOCALES,
+              output: "src/popup/locales/$LOCALE/$NAMESPACE.json",
+              sort: true,
+              useKeysAsDefaultValue: true,
+            },
+          }),
+        ]
+      : []),
```

Comment:
This is where we actually build those translation files. FYI, this leverages the build command for the extension, so `yarn build:extension:translations` will not only build translations, but also build the extension as a side effect